src/clj/harja/palvelin/palvelut/viestinta.clj

```diff
@@ -52,19 +52,19 @@
       (if (empty? viestin-saajat)
         (log/warn (format "Urakalle %s ei löydy FIM:stä yhtään henkiöä, jolle lähettää sähköposti." urakka-sampoid))
         (doseq [henkilo viestin-saajat]
-          (try
-            (sahkoposti/laheta-viesti!
-              email
-              (sahkoposti/vastausosoite email)
-              (:sahkoposti henkilo)
-              (str "Harja: " viesti-otsikko)
-              viesti-body)
-            (catch Exception e
-              (log/error (format "Sähköpostin lähetys FIM-käyttäjälle %s epäonnistui. Virhe: %s"
-                                 (pr-str henkilo) (pr-str e))))))))
+          (sahkoposti/laheta-viesti!
+            email
+            (sahkoposti/vastausosoite email)
+            (:sahkoposti henkilo)
+            (str "Harja: " viesti-otsikko)
+            viesti-body)
+          ))
```

Comment:
Tyylinä ollaan vältetty näitä vain sulkuja sisältäviä rivejä. Eli raakasti vaan nostellaan ne kaikki tonne viimeisen koodia sisältävän rivin perään. paredit huolehtii ettei sulut mene sekaisin.

Reply:
Ok, pistän one-liner-fiksin tähän vielä tulemaan että menee tyyliohjeiden mukaisesti!